mongoimport/json_test.go

```diff
@@ -23,21 +25,21 @@ func TestJSONArrayStreamDocument(t *testing.T) {
 		var jsonFile, fileHandle *os.File
 		Convey("an error should be thrown if a plain JSON document is supplied", func() {
 			contents := `{"a": "ae"}`
-			r := NewJSONInputReader(true, bytes.NewReader([]byte(contents)), 1)
+			r := NewJSONInputReader(true, true, bytes.NewReader([]byte(contents)), 1)
```

Comment:
Pls add a comment to this test function that we're testing legacy extended JSON, because the next reader isn't going to realize what the second "true" argument means.